### PR TITLE
Submitting what I have

### PR DESCRIPTION
**Pull request by sungminchang**
Not sure yet how we're going to serve up different pages to people depending on their identity (presenter or audience).

### CODE REVIEW

File: client/mic.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg height="1000" width="613.77" xmlns="http://www.w3.org/2000/svg"><path d="M0 555.92v-105.896q0 -18.056 12.688 -30.744t30.744 -12.688 30.988 12.688 12.932 30.744v105.896q0 88.816 64.416 152.012t155.184 63.196q90.28 0 154.452 -63.196t64.172 -152.012v-105.896q0 -18.056 12.932 -30.744t30.988 -12.688 30.988 12.688 12.932 30.744v105.896q0 112.728 -74.908 196.908t-186.172 101.26v42.456h136.64q18.056 0 31.232 12.688t13.176 30.256q0 18.056 -12.932 30.744t-31.476 12.688h-361.12q-18.056 0 -30.744 -12.688t-12.688 -30.744q0 -17.568 12.688 -30.256t30.744 -12.688h137.128v-41.968q-73.688 -9.76 -134.444 -51.728t-95.648 -107.116 -34.892 -139.812zm144.448 -4.88v-373.808q0 -66.368 47.58 -113.216t114.924 -46.848q66.856 0 114.192 46.848t47.336 113.216v373.808q0 66.368 -47.336 113.216t-114.192 46.848q-67.344 0 -114.924 -46.848t-47.58 -113.216z"/></svg>
```
Comment (brianpchsu) [inline client/mic.svg:1]: What does this file do? I am also curious about the long number in path.

File: User Story.txt [hunk 2]
```diff
@@ -0,0 +1,7 @@
+Client/Student:
+  Goes to ribbit.com, sees a universal landing page.
+  Signs in with his username and password that is tied to this site.
+    We can also offer the option to sign in through Github/Facebook/whatever.
+  Client updates to reflect the signed in state
+
+Admin/Teacher:
```
Comment (ajkitson) [inline User Story.txt:7]: Do we want this file in the repo?

File: bower.json [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+  "authors": [],
+  "license": "MIT",
+  "ignore": [
+    "**/.*",
+    "node_modules",
+    "bower_components",
+    "client/lib"
+  ],
+  "dependencies": {
+    "angular": "~1.2.18",
```
Comment (ajkitson) [inline bower.json:17]: Why the 1.2.18 version of Angular and not something 1.3.+?

File: client/mic.svg (hunk 1 above)
Comment (ajkitson) [inline client/mic.svg:1]: I believe it's a mic icon, used in audience.html above

File: client/app.js [hunk 4]
```diff
@@ -0,0 +1,28 @@
+var micApp = angular.module('micApp', [
+  'ngRoute',
+  'micControllers'
+]);
+
+micApp.config(['$routeProvider', 
+  function($routeProvider) {
+    $routeProvider
+      .when('/index', {
+        templateUrl: 'index.html',
```
Comment (ajkitson) [inline client/app.js:10]: Is this route ever used?

File: bower.json (hunk 3 above)
Comment (brianpchsu) [inline bower.json:17]: Interesting. Should we all use the latest version of lib in our project (like the lib in package.json)?

Comment (sungminchang) [inline bower.json:17]: Yeah let's update to use the latest version, that was an oversight on my
part. The image is just a stand in for the mic icon, and I think the long
numbers describe the vector path that draws the mic image.

**Pull request closed without merging**